### ISSUE TITLE
php-nginx:7.2 extension "intl" not loaded for TYPO3 9.0

### ISSUE THREAD

**Issue by ad-on-is** [webdevops/Dockerfile#214]
I'm trying to run TYPO3 9.0 with php-nginx:7.2. 

This is the error I get in the "Check Environment" section:
`
PHP extension "intl" not loaded
TYPO3 uses the PHP extension "intl" but it is not loaded in your environment. Change your environment to provide this extension. This extension is used for correct language and locale handling.
`

**Comment by josefglatz** [comment]
I would say, we can enable the PHP extension in general for PHP 7.2 based images (at least for `-dev` images) as also `webdevops/php-apache(-dev)` images are affected

**Comment by josefglatz** [comment]
@ad-on-is You can at least install it by running

```
RUN apt-get -y update \
    && apt-get install -y libicu-dev\
    && docker-php-ext-configure intl \
    && docker-php-ext-install intl
```

in your Dockerfile (as a work around)

**Comment by ad-on-is** [comment]
Yeah sure, but like maybe many others, I use it in a `docker-compose.yml` setup. I like the fact, that I can update my environment by just changing the tag of the webdevops-image, without touching anything else. That's why I think that this is an awesome  project- thx for it ;-)

Right now, I'm not running any TYPO3 9 projects, therefore it's not that urgent. I was just curious about the new version, and when I changed the environment I realised this error, which may be very helpful for you to improve this great project even more :-)
